pkg/controllers/kubernetes/secret.go

```diff
@@ -0,0 +1,53 @@
+package kubernetes
+
+import (
+	"github.com/tektoncd/dashboard/pkg/broadcaster"
+	"github.com/tektoncd/dashboard/pkg/endpoints"
+	logging "github.com/tektoncd/dashboard/pkg/logging"
+	v1 "k8s.io/api/core/v1"
+	k8sinformer "k8s.io/client-go/informers"
+	"k8s.io/client-go/tools/cache"
+)
+
+func NewSecretController(sharedK8sInformerFactory k8sinformer.SharedInformerFactory) {
+	logging.Log.Debug("In NewSecretController")
+	k8sInformer := sharedK8sInformerFactory.Core().V1().Secrets()
+	k8sInformer.Informer().AddEventHandler(cache.ResourceEventHandlerFuncs{
+		AddFunc:    secretCreated,
+		UpdateFunc: secretUpdated,
+		DeleteFunc: secretDeleted,
+	})
+}
+
+func secretCreated(obj interface{}) {
+	logging.Log.Debug("Secret Controller Create")
```

Comment:
`Detected a new secret *name* has been created` would be more useful - assuming we can get the name here

Reply:
Will modify all controller logs to do so.